common/djangoapps/course_modes/api.py

```diff
@@ -0,0 +1,17 @@
+"""
+Python APIs exposed by the course_modes app to other in-process apps.
+"""
+
+from course_modes.models import CourseMode as _CourseMode
```

Comment:
I'm curious -- Why the `_` in front of CourseMode?

Reply:
Since this is an API file, we want the user to only import the objects we define in here. By prefixing the model `CourseMode` with a `_`, it signifies to the user that we the models are internal and we don't want them to access them directly.